### ISSUE TITLE
[1.14.4] Eating Breadroll is causing a crash on SP or kick while on a Server.

### ISSUE THREAD

**Issue by Gbergz** [zmatez/wildnature#69]
Title^

Mod-Version: 2.1.6.4
Forge-Version: 28.2.4
MC: 1.14.4

Kick Message on Server: _Internal Server Error._

SP Crash Report: https://gist.github.com/Gbergz/9e0dada0e8dec33d774269c2549652b2

Any ideas?

**Comment by zmatez** [comment]
Yeah, it will be fixed in 2.1.7. Weird bug, but already fixed in dev(and patron) versions, so be patient ;)

**Comment by Gbergz** [comment]
Awesome! Love the mod. Keep up the great work.

**Comment by Username2-stack** [comment]
It isn't just breadrolls; for me it happens with any food that needs to be crafted besides cakes.
